webapp/client/src/components/FormFieldYesNo.js

```diff
@@ -95,38 +104,30 @@ function FormFieldYesNo({
       hideLabel
       render={() => (
         <YesNo>
-          <FieldLabelForSeparatedFields {...{ label, fieldId, details }} />
+          <FieldLabel {...{ label, fieldId, details }} />
```

Comment:
Wait, I'm confused. I thought this _should_ use `FieldLabelForSeparatedFields`? [In the old world](https://github.com/digitalservice4germany/steuerlotse/blob/650042b632a6fffb9979f2e1b53d36456e779604/webapp/app/templates/components.html#L110), RadioFields (of which YesNo is a subclass) get the "Label" which isn't a label but a legend and that's what `FieldLabelForSeparatedFields` represents.

Reply:
I have changed it because we currently display a label. However, you are correct. It should be a legend and not a label because it "labels" a fieldset.

I think that this should probably be the direction we take for naming the different FieldLabel components (label, legend...)